<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteSearch</class>
 <widget class="QWidget" name="RouteSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(39, 100, 54);
font: 60pt &quot;Oswald&quot;;
border-bottom: 1px solid #000000;
</string>
   </property>
   <property name="text">
    <string>Route Search</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>568</y>
     <width>540</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}
</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>816</y>
     <width>1280</width>
     <height>144</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-bottom-left-radius:5px;
	border-bottom-right-radius:5px;
	border-top: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CurrentTime">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>540</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	font: 30pt &quot;Oswald&quot;;
	background-color: white;
	placeholder-text-color: rgb(156, 163, 175);
	border-radius: 20px;
	border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="CurrentLocation">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>364</y>
     <width>540</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	color: rgb(0, 0, 0);
	font: 30pt &quot;Oswald&quot;;
	background-color: white;
	placeholder-text-color: rgb(156, 163, 175);
	border-radius: 20px;
	border: 1px solid #000000;
}

QComboBox::drop-down {
	width: 0px;
}

QComboBox QAbstractItemView {
    color: rgb(0,0,0);
    font: 18pt 'Oswald';
    background-color: white;
    border-radius: 20px;
    border: 1px solid black;
}</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="Destination">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>636</y>
     <width>540</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	color: rgb(0, 0, 0);
	font: 30pt &quot;Oswald&quot;;
	background-color: white;
	placeholder-text-color: rgb(156, 163, 175);
	border-radius: 20px;
	border: 1px solid #000000;
}

QComboBox::drop-down {
	width: 0px;
}

QComboBox QAbstractItemView {
    color: rgb(0,0,0);
    font: 18pt 'Oswald';
    background-color: white;
    border-radius: 20px;
    border: 1px solid black;
}</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="TimeToggle">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>432</y>
     <width>540</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 30pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}</string>
   </property>
   <property name="text">
    <string>Current Time: OFF</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
